app/Main.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+module Main where
+
+import Options.Applicative
+import Data.Semigroup ((<>))
+
+import BenchShow
+
+verbose :: Parser Bool
+verbose = switch ( long "verbose" <> short 'v' <>
+    help "Provide more details in the report" )
+
+title :: Parser String
+title = strOption (long "title" <> short 't'
+                  <> metavar "Report title"
+                  <> help "Report title, information like plotted field name etc.")
+
+titleField :: Parser TitleAnnotation
+titleField = flag' TitleField
+    ( long "titlefield"
```

Comment:
For CLI we can use the convention to use hyphen to separate words. So a camel case like `TitleField` becomes `--title-field` on CLI.